src/main/java/com/stripe/param/TokenCreateParams.java

```diff
@@ -0,0 +1,2512 @@
+       * The type of entity that holds the account. It can be `company` or `individual`. This field
+       * is required when attaching the bank account to a `Customer` object.
+       */
+      public Builder setAccountHolderType(AccountHolderType accountHolderType) {
+        this.accountHolderType = accountHolderType;
+        return this;
+      }
+
+      /** The account number for the bank account, in string form. Must be a checking account. */
+      public Builder setAccountNumber(String accountNumber) {
+        this.accountNumber = accountNumber;
+        return this;
+      }
+
+      /** The country in which the bank account is located. */
+      public Builder setCountry(String country) {
+        this.country = country;
+        return this;
+      }
+
+      /**
+       * The currency the bank account is in. This must be a country/currency pairing that [Stripe
+       * supports.](docs/payouts)
+       */
+      public Builder setCurrency(String currency) {
+        this.currency = currency;
+        return this;
+      }
+
+      /**
+       * The routing number, sort code, or other country-appropriateinstitution number for the bank
+       * account. For US bank accounts, this is required and should bethe ACH routing number, not
+       * the wire routing number. If you are providing an IBAN for`account_number`, this field is
+       * not required.
+       */
+      public Builder setRoutingNumber(String routingNumber) {
+        this.routingNumber = routingNumber;
+        return this;
+      }
+    }
+
+    public enum AccountHolderType implements ApiRequestParams.Enum {
+      @SerializedName("company")
+      COMPANY("company"),
+
+      @SerializedName("individual")
+      INDIVIDUAL("individual");
+
+      @Getter private final String value;
+
+      AccountHolderType(String value) {
+        this.value = value;
+      }
+    }
+  }
+
+  @Getter
+  public static class CreditCard {
+    @SerializedName("address_city")
+    String addressCity;
+
+    @SerializedName("address_country")
+    String addressCountry;
+
+    @SerializedName("address_line1")
+    String addressLine1;
+
+    @SerializedName("address_line2")
+    String addressLine2;
+
+    @SerializedName("address_state")
+    String addressState;
+
+    @SerializedName("address_zip")
+    String addressZip;
+
+    @SerializedName("card_type")
+    String cardType;
+
+    @SerializedName("currency")
+    String currency;
+
+    @SerializedName("customer")
+    String customer;
+
+    @SerializedName("default_for_currency")
+    Boolean defaultForCurrency;
+
+    @SerializedName("exp_month")
+    String expMonth;
+
+    @SerializedName("exp_year")
+    String expYear;
+
+    @SerializedName("last4")
+    String last4;
+
+    @SerializedName("metadata")
+    Map<String, String> metadata;
+
+    @SerializedName("name")
+    String name;
+
+    @SerializedName("number")
+    String number;
+
+    @SerializedName("number_looks_valid")
+    Boolean numberLooksValid;
```

Comment:
same, this looks like really leaking something

Reply:
Addressed